README.md

```diff
@@ -101,8 +101,8 @@ Dev snapshots of Toree are located at https://dist.apache.org/repos/dist/dev/inc
 of those packages, you can use the following:
 
 ```
-pip install <PIP_RELEASE_URL>
-jupyter toree install
+pip install --upgrade toree
+jupyter toree install --spark_home=/usr/local/bin/apache-spark/
 ```
 
 where `PIP_RELEASE_URL` is one of the `pip` packages. For example:
```

Comment:
this paragraph will now become obsolete.

Reply:
ok,but when i enter the command "pip install <PIP_RELEASE_URL> 
jupyter toree install", jupyter console reports "spark not found" cause jupyter sets spark_home as default,actually i didn't install spark at that path. I think it is a little bit ambiguous

Reply:
@Key2Love I am not disagreeing with your pr, per contrary, it's good, but with the change, we should remove the lines below as they were applying to old contents, no?